packages/server/lib/browsers/firefox.ts

```diff
@@ -382,7 +382,7 @@ export function clearInstanceState (options: GracefulShutdownOptions = {}) {
 }
 
 export async function connectToNewSpec (browser: Browser, options: BrowserNewTabOpts, automation: Automation) {
-  await firefoxUtil.connectToNewSpec(options, automation, browserCriClient)
+  await firefoxUtil.connectToNewSpec(options, automation, browserCriClient!)
```

Comment:
could we just return?

Reply:
I prefer it doesn't return since it more clearly communicates that the return value is not used. The only reason this line changed is that I added the `!` to satisfy TypeScript after I changed the type from `any` to `BrowserCriClient | undefined`.

Reply:
> I prefer it doesn't return since it more clearly communicates that the return value is not used
thats a good way to look at it. 👍🏻 I like that logic